src/applications/yellow-ribbon/containers/SearchForm/index.jsx

```diff
@@ -201,20 +201,20 @@ export class SearchForm extends Component {
 
           <div>
             {/* Unlimited Contribution Amount */}
-            <Checkbox
+            <VaCheckbox
               checked={contributionAmount === 'unlimited'}
               name="contributionAmount"
               label="Only show schools that provide maximum funding (tuition that's left after your Post-9/11 GI Bill)"
-              onValueChange={onCheckboxChange('contributionAmount')}
+              onVaChange={() => onCheckboxChange('contributionAmount')}
```

Comment:
The `onVaChange` event details is returned as `event.detail.checked` (boolean) but the implementation does not seem to want to know true/false (just the string "contributionAmount") so I have kept that the same.